scripts/set-user.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-SESSIONS_DB_NAME="sessions.db"
+SESSIONS_DB_NAME="trailhead.db"
```

Comment:
I know this was in the other commit so you need to rebase this against the current main and then I can take a look at it. Dara is also making changes so there's more change happening here than you might expect.

As a general rule, I don't care whether your PRs are 1 commit or many, but they do need to be isolated changes (and not have any previously-merged changes in them, because that can leads to some annoying detangling)